packages/conf-git/conf-git.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://git-scm.com"
+authors: "Linus Torvalds"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "git+https://github.com/ocaml/opam-repository.git"
+license: "GPL-2+"
+build: [["which" "git"]]
```

Comment:
Does this mean that it depends on `conf-which`?

Reply:
yes.

Reply:
fixed.